### DISCUSSION TITLE
Typography: custom h3::before in a ordered list

### DISCUSSION THREAD

**Discussion by ahoiroman** [tailwindlabs/tailwindcss#4858] [Help]
Hello everybody,

I customized the `h3` like this:

```
h3: {
    color: theme('colors.red.500'),
    fontFamily: `${theme('fontFamily.mono')}`,
    fontWeight: '400',
    fontSize: '1rem',
},
```

Now I got an `ol` like this:

```
<li>
<h3>Lorem ipsum</h3>
<ol>
  <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
  </li>
  <li>diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
  </li>
  <li>et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing
  </li>
</ol>
</li>
```

Now this is happening:

<img width="602" alt="Bildschirmfoto 2021-07-01 um 14 33 59@2x" src="https://user-images.githubusercontent.com/23051150/124125014-6bd5be00-da79-11eb-9956-f2fa1602f7b7.png">

As you can see, the color of `h3::before` is not being adjusted to `colors.red.500`;

If I now add

```
'ol > li::before': {
    content: 'counter(list-item, var(--list-counter-style, decimal)) "."',
    position: 'absolute',
    fontWeight: '300',
    fontSize: '0.875rem',
    color: theme('colors.red.500'),
},
```

this is happening:

<img width="594" alt="Bildschirmfoto 2021-07-01 um 14 35 50@2x" src="https://user-images.githubusercontent.com/23051150/124125247-accdd280-da79-11eb-94bb-c894cb28b863.png">

I just want the `1.` before `Lorem ipsum` to be red, too and the other ones not to be red ;-)

**Comment by olets** [comment]
Have you tried adding a `color` to the `h3::before` styles, like you did to the `ol > li::before` styles in the second example?

Your example markup is missing the top level `ol` so if you tried `ol > h3::before` that would not target your `h3`.

And though it might be necessary in your full real code, in the minimal example you should be able to use a less specific selector than `ol > …`

You might already know this: for general CSS questions like this, https://stackoverflow.com/questions/tagged/css is a good place to ask if you want an answer fast.
